### PR TITLE
Add contact information to about.html.

### LINKED ISSUE #357: General contact information(?)
As far as I know there's no "general" contact information or any place from the website from which you can access the Silent Selene Discord server. There's only _privacy@silentselene.net_ on the Privacy Policy page. Should there be some e-mail to let outsiders contact developers about general questions?

When discussing HRtP default lives rule with Karisa, there was no general contact method for her to get in touch with you guys - only with individual developers (https://www.shrinemaiden.com/index.php?topic=46.msg28127#msg28127)

### PR DESCRIPTION
**Pull request by n-rook**
Fixes #357

image

### CODE REVIEW

File: project/thscoreboard/replays/templates/replays/docs/about.html [hunk 1]
```diff
@@ -16,4 +16,11 @@ <h2>What is Touhou?</h2>
         You can buy most of the official Touhou games on <a href="https://store.steampowered.com/search/?developer=%e4%b8%8a%e6%b5%b7%e3%82%a2%e3%83%aa%e3%82%b9%e5%b9%bb%e6%a8%82%e5%9b%a3">Steam</a>, <a href="https://www.dlsite.com/maniax/circle/profile/=/maker_id/RG46179.html">DLSite</a> or many other online stores. A few are not sold as online downloads, but can be purchased as CDs.
     </p>
     {% endblocktranslate %}
+    {% blocktranslate %}
+    <h1>Contact Us</h1>
+    <p>
+        The best way to contact the Silent Selene team or discuss it is to join us on Discord; you can find our Discord server by joining "The Gap", the public Discord server that serves as a directory for the Touhou community, and searching it for "Silent Selene". You can also send feedback by sending an email to <a href="mailto:webmaster@silentselene.net">webmaster@silentselene.net</a>.
```
Comment (jgdhs27) [inline project/thscoreboard/replays/templates/replays/docs/about.html:22]: I don't like this. could we at least link to "The Gap"? I hadn't heard of it until now.

**Comment by n-rook** [comment]
ah fuck it, I hate it too, I just was worried we'd get bots on the discord server if I linked it directly. but I'll just do it anyway, we can always revoke the invite link and change it later if it becomes an issue

File: project/thscoreboard/replays/templates/replays/docs/about.html (hunk 1 above)
Comment (n-rook) [reply project/thscoreboard/replays/templates/replays/docs/about.html]: Done, I just linked our discord

**Review by jgdhs27 — APPROVED** [review]

**Comment by n-rook** [comment]
woot

**Pull request merged**